app/pages/sections/catalogue-solutions/order-item/routesHelper.js

```diff
@@ -16,7 +17,7 @@ export const getPageData = async ({
 
     const selectedPrice = await getSelectedPrice({ selectedPriceId, accessToken });
     const solutionName = (await getSolution({ solutionId, accessToken })).name;
-    const formData = { price: selectedPrice.price };
+    const formData = { price: formatDecimal(selectedPrice.price) };
```

Comment:
yes this feels like the right place to do it.

Think we also need to do the same when we get the price back from ordapi

Reply:
ah yeah hadn't spotted this was only newsolution